src/hotspot/os/linux/memMapPrinter_linux.cpp

```diff
@@ -27,58 +27,291 @@
 
 #include "runtime/os.hpp"
 #include "nmt/memMapPrinter.hpp"
+#include "utilities/align.hpp"
+#include "utilities/powerOfTwo.hpp"
```

Comment:
This include section is not sorted correctly. It would also be good to add a blank line between the hotspot includes and the system includes.